### PR TITLE
Update desktop-apps.md

### PR DESCRIPTION
**Pull request by smolinari**
Add Quasar for Electron development

Scott

### CODE REVIEW

File: docs/guide/ecosystem/desktop-apps.md [hunk 1]
```diff
@@ -84,6 +84,22 @@ Electron and NW.js share many features: they are both built on top of Chromium a
 - During the build process NW.js builds the whole Chromium; Electron uses [libchromiumcontent](https://github.com/electron/libchromiumcontent) to access Chromium's Content API
 - NW.js has [built-in support](http://docs.nwjs.io/en/latest/For%20Users/Advanced/Protect%20JavaScript%20Source%20Code/) for compiling your source code to protected binaries. Electron is packaging its applications with [asar](https://github.com/electron/asar), which contains the applications' unprotected source code.
 
+## Quasar Framework
+
+[Quasar Framework](https://quasar.dev) is a framework which allows for cross-platform application development. Design your apps with general components and code and use it all for any platform, including Electron for Desktop applications. Quasar's CLI allows you to build your Electron desktop application with one simple command.
```
Comment (dobromir-hristov) [inline docs/guide/ecosystem/desktop-apps.md]: Could you rephrase the part with `Design your apps with` as it has too much `and` words next to each other. Reads a bit weird. 

I think we should emphasize on the fact that you have an enormous collection of awesome UI components, that are very well coupled AND using the CLI you can build them to an Electron app.

**Comment by smolinari** [comment]
Thanks for the merge!

Scott
